From 2e176e3d10535e4fdaa6ecc919538f66f4a2930d Mon Sep 17 00:00:00 2001
From: Andy Nichols <andy.nichols@qt.io>
Date: Mon, 11 Jul 2016 09:18:33 +0200
Subject: QWidgetBackingStore: Fix build with QT_NO_OPENGL

A define moved out of the #ifndef QT_NO_OPENGL scope.

Change-Id: I41c943b97f165b5171581e01f3a4b2d3fb25747e
Reviewed-by: Laszlo Agocs <laszlo.agocs@qt.io>

Upstream-Status: Backport
Signed-off-by: Johannes Pointner <johannes.pointner@br-automation.com>
---
 src/widgets/kernel/qwidgetbackingstore.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/widgets/kernel/qwidgetbackingstore.cpp b/src/widgets/kernel/qwidgetbackingstore.cpp
index 32aabce..242e625 100644
--- a/src/widgets/kernel/qwidgetbackingstore.cpp
+++ b/src/widgets/kernel/qwidgetbackingstore.cpp
@@ -67,7 +67,9 @@ QT_BEGIN_NAMESPACE

 extern QRegion qt_dirtyRegion(QWidget *);

+#ifndef QT_NO_OPENGL
 Q_GLOBAL_STATIC(QPlatformTextureList, qt_dummy_platformTextureList)
+#endif

 /**
  * Flushes the contents of the \a backingStore into the screen area of \a widget.
--
cgit v1.0-4-g1e03

